js/index.js

```diff
@@ -1,32 +1,23 @@
-window.onload = adjustIframeHeight
-window.sizer = adjustIframeWidth
-
-require('lity')
-
-var $ = document.querySelectorAll.bind(document)
-
-function displayIframe(iFrame){
-  iframe.style
-}
-
-
-function adjustIframeWidth(size){
-  var iFrames = $('iframe')
-
-  ;[].map.call(iFrames, (frame)=>{
-    debugger
-    frame.style.width = size
-    return
-  })
-}
-
-function adjustIframeHeight(){
-  var iFrames = $('iframe')
-
-  ;[].map.call(iFrames, (frame)=>{
-    var realHeight = frame.contentWindow.document.body.offsetHeight
-    frame.style.height = realHeight+"px"
-    return
-  })
-
-}
\ No newline at end of file
+var $ = require('jquery');
+require('lity');
+
+$(document).ready(function () {
+  var lastbreakPoint;
+
+  $('.resize-iframe-toggle').click(function () {
```

Comment:
@soundyogi was unsure how to call the classes or wheter to reuse ck-labsizer__item for this.. I once used the convention to prefix classes that bind js code with "fx-"..